tests/unit/schema/printers/test_extract_schema.py

```diff
@@ -0,0 +1,99 @@
+from collections import defaultdict
```

Comment:
Can you follow the pattern for the other schema printers? 
https://github.com/nodestream-proj/nodestream/blob/77579a2fe237890c7152776ce50674cba9d5b2e7/tests/unit/schema/printers/test_graphql_schema_printer.py